src/ws/cockpitauth.c

```diff
@@ -60,6 +60,8 @@ guint cockpit_ws_process_idle = 600;
 
 static guint sig__idling = 0;
 
+static gboolean gssapi_not_avail = FALSE;
```

Comment:
Because we don't know if it's available. In theory there is a tristate here:

 1. Available
 2. Unknown
 3. Not available

We only care about the last two cases of the tristate.